config/deploy.rb

```diff
@@ -8,7 +8,7 @@
 set :deploy_to, '/home/apps/t4c'
 
 set :rvm_type, :user
-set :rvm_ruby_version, '3.2.6'
+set :rvm_ruby_version, File.read("#{File.dirname(__FILE__)}/../.ruby-version").strip
```

Comment:
This style recommended/documented here: https://github.com/capistrano/rbenv?tab=readme-ov-file#usage